sqlobject/src/test/java/org/jdbi/v3/sqlobject/locator/TestSqlLocator.java

```diff
@@ -32,7 +32,7 @@
     public PgDatabaseRule dbRule = new PgDatabaseRule().withPlugin(new SqlObjectPlugin());
 
     @Test
-    public void testLocateConfigDriven() {
+    public void testLocateConfigDriven() throws Exception {
```

Comment:
type inference changed somewhere along the way, and this no longer compiles in 11??

Reply:
You're saying JDK 11 breaks type inference on our exception transparency stuff? Well.. shit

Reply:
I'm not sure exactly what the break is.  Only this test case required fixing up.
We could submit a question to the jdk mailing lists if we can extract a small reproduction case.  I'm not sure what the impact will be yet, I only just got the build to go green...